modules/xmpp/TraceablePeerConnection.js

```diff
@@ -547,7 +547,9 @@ TraceablePeerConnection.prototype.createAnswer
                         dumpSDP(answer));
                 }
 
-                if (!this.session.room.options.disableRtx && !RTCBrowserType.isFirefox()) {
+                if (typeof this.session.room.options.disableRtx !== 'undefined'
```

Comment:
and actually, isn't the 'undefined' case eliminated here by the logic you added in moderator.js?